leases/tests/test_lease_queries.py

```diff
@@ -168,7 +178,7 @@ def test_query_berth_lease(superuser_api_client, berth_lease, berth_application)
 
 
 @pytest.mark.parametrize(
-    "api_client", ["api_client", "user_api_client", "staff_api_client"], indirect=True
+    "api_client", ["api_client", "user", "harbor_services"], indirect=True
```

Comment:
should `test_query_berth_lease_invalid_id` down below use `superuser_ api_client`? 🤔

Reply:
Terrrrrrific catch. 👍